api/v2/api.go

```diff
@@ -62,7 +63,10 @@ type API struct {
 	queues      queues
 	service     string
 	version     string
-
+	swarm       struct {
+		endpoint1 *swampi.Swampi
+		endpoint2 *swampi.Swampi
+	}
```

Comment:
Why is there a hard limit on two endpoints?

Reply:
no real reason. at the moment we only need 2, however I've gone ahead and structure it in such a way that you can use more than 2 endpoints if the config package is adjusted to support it